azure_functions_worker/functions.py

```diff
@@ -78,6 +78,15 @@ def get_return_binding(binding_name: str,
 
         return return_binding_name
 
+    @staticmethod
+    def validate_binding_route(func_name: str, binding: BindingInfo,
+                               func_type: str):
+        if hasattr(binding, 'route') and binding.route.startswith(
+                '/') and func_type == 'function':
+            raise FunctionLoadError(
+                func_name,
+                f'Invalid route name. {binding.route}')
```

Comment:
Can you be verbose here? And also an action plan for the user - what should they change to get out of this error?

Reply:
Fixed. Added a more verbose message which should tell the user that the trigger name cannot start with /